easybuild/easyconfigs/c/cas-offinder/cas-offinder-2.4-foss-2018b.eb

```diff
@@ -0,0 +1,42 @@
+# easybuild easyconfig
+#
+# 2019.07.01 John Dey jfdey@fredhutch.org fizwit@github.com
+# Notes: requires OpenGL. pocl is providing opencl libraries, but might be better
+# with fosscuda
+easyblock = 'CMakeMake'
+
+name = 'cas-offinder'
+version = '2.4'
+
+homepage = ''
+description = """Cas-OFFinder is OpenCL based, ultrafast and versatile program that searches
+ for potential off-target sites of CRISPR/Cas-derived RNA-guided endonucleases (RGEN)."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
```

Comment:
@fizwit From the listed dependencies this software could be built with the corresponding `GCC` toolchain instead. Is there some library in `foss` that is required?